composer.json

```diff
@@ -481,13 +507,6 @@
       "@php bin/console bundle:dump",
       "@admin:generate-entity-schema-types",
       "@npm:admin run dev"
-    ],
-    "watch:storefront": [
```

Comment:
Wouldn't it be good to at least leave it as an alias? Just to avoid frustration for people not knowing about this change 🙂